crates/derive/README.md

```diff
@@ -2,5 +2,21 @@
 
 > **Notice**: This crate is a WIP.
 
-A `no_std` compatible implementation of the OP Stack's
-[derivation pipeline](https://specs.optimism.io/protocol/derivation.html#l2-chain-derivation-specification).
+A `no_std` compatible implementation of the OP Stack's [derivation pipeline][derive].
+
+[derive]: (https://specs.optimism.io/protocol/derivation.html#l2-chain-derivation-specification).
+
+## Features
+
+The most up-to-date feature list will be available on the [docs.rs `Feature Flags` tab][ff] of the `kona-derive` crate.
+
+Some features include the following.
+- `serde`: Serialization and Deserialization support for `kona-derive` types.
+- `k256`: [secp256k1][k] public key recovery support.
```

Comment:
Also would note _why_ we allow for disabling the native key recovery, to allow for an accelerator within the host program in "offline" mode